packages/server/src/api/controllers/row/external.ts

```diff
@@ -39,8 +39,8 @@ export async function handleRequest<T extends Operation>(
 
 export async function patch(ctx: UserCtx<PatchRowRequest, PatchRowResponse>) {
   const tableId = utils.getTableId(ctx)
+  const oldRow = ctx.request.body
```

Comment:
This will contain the new row - not the old row, the old row will only be retrievable by performing a `db.get(rowId)` at the start here, then putting it through `outputProcessing`.

Reply:
Sorry yes, I need to update that to follow the pattern used in internal.